static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -1,42 +1,124 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
 
-import EventEntry from 'sentry/components/events/eventEntry';
-import {Entry, Event} from 'sentry/types/event';
-import useOrganization from 'sentry/utils/useOrganization';
-import {useRouteContext} from 'sentry/utils/useRouteContext';
+import Type from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/type';
+import {
+  onlyUserActions,
+  transformCrumbs,
+} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import {
+  Panel as BasePanel,
+  PanelBody as BasePanelBody,
+  PanelHeader as BasePanelHeader,
+  PanelItem,
+} from 'sentry/components/panels';
+import ActionCategory from 'sentry/components/replays/actionCategory';
+import PlayerRelativeTime from 'sentry/components/replays/playerRelativeTime';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import {relativeTimeInMs} from 'sentry/components/replays/utils';
+import space from 'sentry/styles/space';
+import {Crumb, RawCrumb} from 'sentry/types/breadcrumbs';
+import {Event} from 'sentry/types/event';
 
 type Props = {
-  entry: Entry | undefined;
+  crumbs: RawCrumb[];
   event: Event | undefined;
 };
 
-function UserActionsNavigator({event, entry}: Props) {
-  // TODO(replay): New User Actions widget replaces this breadcrumb view
+function UserActionsNavigator({event, crumbs}: Props) {
+  const {setCurrentTime} = useReplayContext();
+  const [currentUserAction, setCurrentUserAction] = useState<Crumb>();
 
-  const {routes, router} = useRouteContext();
-  const organization = useOrganization();
+  useEffect(() => {
+    currentUserAction?.timestamp &&
+      startTimestamp &&
+      setCurrentTime(relativeTimeInMs(currentUserAction.timestamp, startTimestamp));
+  }, [currentUserAction]);
```

Comment:
This `useEffect` block is going to reset the currentTime on initial render, which will break the `?t=30` query param.

Instead just put the `setCurrentTime` call inside the onclick handler below then it only runs after a click:

```
onClick={() => {
  setCurrentUserAction(item); 
  setCurrentTime(relativeTimeInMs(item.timestamp, startTimestamp));
}}
```